ios/ClevertapSignedCallReactNative.swift

```diff
@@ -124,23 +124,30 @@ class CleverTapSignedCall: RCTEventEmitter {
     }
     
     @objc func callStatus(notification: Notification) {
+        let callDetails = notification.userInfo?["callDetails"] as? SCCallStatusDetails
+        
+        let status: SCCallStatus? = callDetails?.status
+        let callDirection: String = callDetails?.callDirection.rawValue ?? ""
         let callEvent = notification.userInfo?["callStatus"] as? SCCallStatus
+        let calleeCuid: String = callDetails?.callDetails.calleeCuid ?? ""
+        let callerCuid: String = callDetails?.callDetails.callerCuid ?? ""
+        let initiatorImage: String = callDetails?.callDetails.initiatorImage ?? ""
+        let receiverImage: String = callDetails?.callDetails.receiverImage ?? ""
```

Comment:
React native model class supports nullable `initiatorImage` and `receiverImage`. 
So, should we consider passing them as null from iOS similar to Android?

Reply:
Fixed